template-helpers/dashboard.js

```diff
@@ -0,0 +1,66 @@
+"use strict";
+
+const { LocaleUtils } = require("./../locale-utils");
+const { makeBreachCards } = require("./breaches");
+
+const getBreachesForEachEmail = (args) => {
+  const verifiedEmails = args.data.root.verifiedEmails;
+  const locales = args.data.root.req.supportedLocales;
+
+  verifiedEmails.forEach(email => {
+    const breachCards = makeBreachCards(email.breaches, locales);
+    email.breaches = breachCards;
+  });
+  return verifiedEmails;
+};
+
+
+const getUserPreferences = (args) => {
+  const unverifiedEmails = args.data.root.unverifiedEmails;
+  const verifiedEmails = args.data.root.verifiedEmails;
+  const primaryEmail = verifiedEmails.shift();
+
+  const locales = args.data.root.req.supportedLocales;
+  args.data.root.preferences = true;
+
+  const emailAddresses = {
+    primary: {
+      subhead: "Firefox Account Email - Primary",
+      email_addresses: [ primaryEmail ], // put in array for template looping
+    },
+    secondary: {
+      subhead: "Other Monitored Emails",
+      email_addresses: verifiedEmails,
+    },
+    unverified: {
+      subhead: "Email Verification Required",
+      email_addresses: unverifiedEmails,
+    },
+  };
+
+  const communicationOptions = [
+    {
+      optionLabel: "Email me if one of my email addreseses below appears in a data breach.",
+      labelString: LocaleUtils.fluentFormat(locales, "comm-opt-0", {primaryEmail: primaryEmail.email}),
+      optionId: "0",
+    },
+    {
+      optionLabel: "Send all breach alerts to ${primary email}",
```

Comment:
Should this use double quotes or template backticks?
Wasn't sure if this was used elsewhere, or if the `${primaryEmail}` should be... actually, what is `${primary email}`? Seems a bit odd for a variable name.

Reply:
This is a description of the option, but the key name is totally confusing, sorry. Updating this to `optionDescription: "Send all breach alerts to subscriber's primary email address."`